### ISSUE TITLE
"Close icon display location" dropbox rendering cut off

### ISSUE THREAD

**Issue by ferdnyc** [BiglySoftware/BiglyBT#767]
Along the lines of other GTK3 text size/rendering adjustments mentioned in recent beta release notes, at some point relatively recently (after the 1.7.0.0 release), the longest option in the dropbox for the Options > Interface > Display > Side Bar setting "Close icon display location" became slightly too wide for the containing dropbox (or the dropbox became too narrow for the string):

![screenshot from 2018-12-25 10-30-53](https://user-images.githubusercontent.com/538020/50424383-d9615d80-0830-11e9-920e-cfb182f094bc.png)

### "System" info...
```
Java 1.8.0_191 (64 bit)
  Oracle Corporation
/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.191.b12-11.fc29.x86_64/jre

SWT v4922, gtk/3.24.1, zoom=100, dpi=96
Linux v4.19.9-300.fc29.x86_64, amd64 (64 bit)
B1.7.0.1_B30/4 az3
```
(Fedora Linux 29; LXDE desktop — but same issue observed under Gnome Shell)

**Comment by parg** [comment]
Should be fixed in next beta

**Comment by ferdnyc** [comment]
Yarp. Fixed in `1.7.0.1_B31`.
